app/models/spree/vendor.rb

```diff
@@ -16,6 +16,8 @@ class Vendor < Spree::Base
       validates_associated :image
     end
 
+    validates :notification_email, email: true, allow_blank: true
```

Comment:
@aplegatt let's also add a logic to set the notification_email with the first user assigned to vendor email